accessing-data/status-webhooks.mdx

```diff
@@ -29,3 +29,357 @@ A status of `records_submitted` means that all records from an Import are curren
 
 3. You update the status of the Import
 
+
+## Status Webhook Implementation Example
+
+Find the sample code below for implementing a status webhook that will be listening to Import status changes so you can access the data through Fuse API and manipulate it how you need.
+
+For this example, we call the Fuse API to retrieve the records for that Import when the status is `records_submitted`. We then save the data and call the Fuse API again to update the Import status inside Fuse.
+
+<CodeGroup>
+
+```typescript Node.js (Nest.js / Express.js)
+import { Request, Response, Router } from 'express';
+import crypto from 'crypto';
+import axios from 'axios';
+
+const router = Router();
+
+router.put("/status-webhook", async (req: Request, res: Response) => {
+  // Retrieve the signature from the headers
+  const receivedSignature = req.headers["http_x_fuse_signature_256"];
+
+  // Compute a signature using the payload and your secret key
+  const secretKey = 'your_secret_key';
+  const expectedSignature = crypto
+    .createHmac("sha256", secretKey)
+    .update(JSON.stringify(req.body))
+    .digest("hex");
+
+  // Compare the signatures
+  if (receivedSignature !== expectedSignature) {
+    return res.status(401).send("Mismatched signatures");
+  }
+
+  // Data is stored in the Fuse servers, let's consume it!
+  if (req.body.status === "records_submitted") {
+    const importSlug = req.body.import_id;
+    const importRecords = await axios.get<any>(
+      `https://fuse.flatirons.com/api/v1/imports/${importSlug}/records`,
+      {
+        headers: { Authorization: "YOUR_API_TOKEN" },
+      }
+    );
+
+    const recordsToStore = importRecords.data;
+
+    // Do what you need with the records. In this example, let's store in our database
+    // Use the implementation for your database! This is just an example
+    await importRepository.createRecords(recordsToStore);
+
+    // Stored and processed! Let's update the Import status inside Fuse
```

Comment:
This seems to have a bug.
I mentioned this with Mike, but apparently, we can't update the status to `records_processed` using the API. Not sure if this was corrected or not, but when trying to do the API call in this example, we get this error message:
`data: { message: 'Import already processed.' }`